homeassistant/components/enigma2/media_player.py

```diff
@@ -131,7 +131,9 @@ def state(self):
         """Return the state of the device."""
         if self.e2_box.is_recording_playback:
             return STATE_PLAYING
-        return STATE_OFF if self.e2_box.in_standby else STATE_ON
+        if self.e2_box.in_standby or self.e2_box.is_offline:
```

Comment:
If a device is offline, the entity should become unavailable (thus not `off`).

This can be implemented using the `available` property.

Reply:
I don't have the knowledge to introduce it,
maybe someone else will try to correct https://github.com/home-assistant/core/issues/42258 or please close this PR

Reply:
it is a bit incomprehensible for me because in hass we have state STATE_STANDBY 
and currently if the tuner is indeed in this state we return STATE_OFF 
and for the tuner (real)state off we need to return a value other than STATE_OFF .... hmmm